src/libraries/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/Rss20FeedFormatterTests.cs

```diff
@@ -449,7 +449,6 @@ public void Write_HasFeed_SerializesExpected()
                 writer.WriteEndElement();
             });
 
-            var genericFormatter = new Rss20FeedFormatter<SyndicationFeed>(feed);
             CompareHelper.AssertEqualWriteOutput(expectedFull, writer => formatter.WriteTo(writer));
```

Comment:
This line should be testing 'genericFormatter' so the line above should be left in.

Reply:
Fixed!

The line `CompareHelper.AssertEqualWriteOutput(expectedFull, writer => feed.SaveAsRss20(writer));` is now duplicated for both formatter and genericFormatter, though it isn't clear to me if there's a change to be made there or not.

Let me know what you think. Thanks!